### ISSUE TITLE
Remat Sound ends too early.

### ISSUE THREAD

**Issue by Noname12345GER** [MattJeanes/TARDIS#260]
I noticed that flight Sounds stop very early during Teleport as the door doesn't open if the sound stops.

**Comment by MattJeanes** [comment]
What?

**Comment by theangelcastiel** [comment]
Could you record a video to help him better understand?

**Comment by Noname12345GER** [comment]
Matt pls.

If you Rematerialize while you are i  the interior, you can hear the flight Sounds right? They are supposed to end when Rematerialization is completed. If the Sounds are completed, i can't open the door unless i wait for additional 10 seconds.

**Comment by MattJeanes** [comment]
Can you make a video showing this problem? It seems to work fine for me

**Comment by Noname12345GER** [comment]
Im unable to capture Videos with my Setup. And If I do, a 30 seconds Video has a size of 500 MB

**Comment by Noname12345GER** [comment]
Found the cause:

The new DeLorean Teleport.

If you selected a Position, the Demat Sound doesn't Start.
Also, if you go pilot, press remat and leave pilot, you will only hear the regular flight Sound but because the Remat Sound didn't Start, the regular flight Sound wenns just there as it should.

Sounds should be played after pilot made an action.

**Comment by MattJeanes** [comment]
Ok, I have another issue open for that bug so I'll close this

**Comment by Noname12345GER** [comment]
Where?

**Comment by theangelcastiel** [comment]
#245

**Comment by Noname12345GER** [comment]
Thanks.
